grails-app/controllers/org/pih/warehouse/api/InvoiceApiController.groovy

```diff
@@ -192,7 +192,7 @@ class InvoiceApiController {
         render([data: invoice?.toJson()] as JSON)
     }
 
-
+    @Deprecated
```

Comment:
can you add a comment to all these deprecated methods explaining why they're being deprecated? (Or are you planning on fully removing them in a later commit?)

Reply:
Sure, will do it